src/cli/bunx_command.zig

```diff
@@ -315,6 +315,10 @@ pub const BunxCommand = struct {
             true,
         );
 
+        if (!ctx.debug.loaded_bunfig) {
+            try bun.CLI.Arguments.loadConfig(ctx.allocator, null, ctx, comptime .BunxCommand);
+        }
+
```

Comment:
right now you cant `--config=./path/bunfig.toml`, but I don't think many people using `bunx` would need that - its more for global `--bun`.